libminifi/include/core/state/Value.h

```diff
@@ -87,6 +88,15 @@ class Value {
     type_id = std::type_index(typeid(T));
   }
 
+  virtual bool getValue(uint32_t &ref) {
+    const auto negative = string_value.find_first_of('-') != std::string::npos;
```

Comment:
You don't really need the variable here, simply put this in the "if"

Reply:
@arpadboda: The name helps readability and since it's `const` it doesn't increase complexity. (by potentially introducing extra states)

Reply:
In my opinion the variable just decreases readability here, but this is really minor, I'm fine with the current way as well. 

Although it's simply wrong. 
It's going to fail in case the '-' char is after the digits (like "3-"), which std::stoull handles. 
Either let it go or go strict and apply isdigit check for all.

I will create a Jira for this, should be done in scope of a separate PR.

Reply:
Will keep the current impl for now